packages/common/src/operations.graphql

```diff
@@ -218,7 +218,7 @@ query requisitions($params: RequisitionListParameters) {
         id
         comment
         orderDate
-        otherPartyReference
+        theirReference
```

Comment:
more consistent

Reply:
I prefer it - but isn't `otherPartyReference` more consistent when we have `otherPartyName`?

Reply:
😢 

`theirReference` is on invoices so consistent in relation with that. Maybe it should be `otherPartyReference` for all of them 🤔